src/tactic/ui/panel/collection_wdg.py

```diff
@@ -651,38 +686,122 @@ def get_display(my):
 
         collection = Search.get_by_search_key(my.collection_key)
 
-
         top = my.top
 
         my.kwargs["scale"] = 75;
         my.kwargs["show_scale"] = False;
-
-
+        my.kwargs["expand_mode"] = "plain"
         from tile_layout_wdg import TileLayoutWdg
         tile = TileLayoutWdg(
             **my.kwargs
         )
+        parent_keys = my.kwargs.get("parent_keys")
+        has_parent=False
+        if parent_keys:
+            has_parent = True
 
         path = my.kwargs.get("path")
         if collection and path:
             title_div = DivWdg()
             top.add(title_div)
             title_div.add_style("float: left")
-            title_div.add_style("margin: 15px 0px")
+            title_div.add_style("margin: 15px 0px 15px 30px")
 
-            icon = IconWdg(name="View Collection", icon="BS_FOLDER_OPEN")
-            title_div.add(icon)
-            icon.add_style("margin-right: 10px")
+            asset_lib_span_div = SpanWdg()
+            title_div.add(asset_lib_span_div)
 
+            icon = IconWdg(name="Asset Library", icon="BS_FOLDER_OPEN")
+            
+            asset_lib_span_div.add(icon)
+
+            asset_lib_span_div.add(" <a><b>Asset Library</b></a> ")
+            
             path = path.strip("/")
             parts = path.split("/")
 
-            for part in parts:
+            for idx, part in enumerate(parts):
                 title_div.add(" / ")
-                title_div.add(" <a class='spt_colleciton_link'><b>%s</b></a> " % part)
+                # the last spt_collection_link does not need a search_key
+                if has_parent and (idx is not len(parts) - 1):
+                    # reverse order of parent_keys to add to attr of each spt_collection_link
+                    search_key = parent_keys[len(parts)-idx-2]
+                    title_div.add(" <a class='spt_collection_link' search_key=%s><b>%s</b></a> " % (search_key, part))
+                else:
+                    title_div.add(" <a class='spt_collection_link'><b>%s</b></a> " % part)
                 title_div.add_style("margin-top: 10px")
-                title_div.add_style("margin-left: 20px")
-            #title_div.add("/ %s" % collection.get_value("name") )
+
+
+            # Adding behavior to collections link
+
+            parts = my.kwargs.get("search_type").split("/")
+            collection_type = "%s/%s_in_%s" % (parts[0], parts[1], parts[1])
+
```

Comment:
It should do a check here to see if the collection_type .e.g workflow/asset_in_asset actually exists